tests/integration/informer_mock_test.go

```diff
@@ -0,0 +1,29 @@
+package biz
+
+import (
+	"context"
+	"github.com/docker/docker/api/types"
+)
+
+const (
+	containerID = "my-container"
+	pid         = 666
+)
+
+type InspectorMock struct{}
```

Comment:
Nit: I noticed the struct name and the file names don't match. Should it be inspector or informer?

Reply:
It should be Inspector, I guess I had a short circuit 😕  I'll change it

Reply:
done